ballerina/http_request_context.bal

```diff
@@ -21,7 +21,7 @@ import ballerina/lang.value;
 public isolated class RequestContext {
     private final map<value:Cloneable|isolated object {}> attributes = {};
 
-    public isolated function add(string 'key, value:Cloneable|isolated object {} value) {
+    public isolated function set(string 'key, value:Cloneable|isolated object {} value) {
```

Comment:
Let's document this public methods

Reply:
Fixed in [`686c856`](https://github.com/ballerina-platform/module-ballerina-http/pull/746/commits/686c8561f606ded81f0af6e6aeb66fa1b09a20b6)